### ISSUE TITLE
Error cheking Whateverfile.hs: Error: Illegal argument: character must be non-negative

### ISSUE THREAD

**Issue by agocorona** [dramforever/vscode-ghc-simple#88]
It compiles some package, but when I open other package that depends on the former, the file is compiled according with the log but at the end produces that error -the one of the title- and the results are not reflected in the problems tab.

That it is probably a bug in VSCode, but I use the latest version

https://github.com/VSCodeVim/Vim/issues/5748

Some workaround for this?

**Comment by dramforever** [comment]
Can you clarify what your issue is? I couldn't figure out what the title has to do with the description.

Can you provide steps to reproduce the problem?

For the issue described in the title, I thought I have fixed this in 20ab62c50b0d33ba7755f6ee59e824b34e9d49ed, which was released a few days ago in v0.1.23, but there could be other shenanigans going on. A file/repo that can reproduce this problem would be great.

**Comment by agocorona** [comment]
Can I load that last release by updating the extension uninstall-> install or there are some other steps necessary?

The title is the error produced, sorry for not having explicitly mentioned it.

**Comment by dramforever** [comment]
You can select the 'extensions' tab on the left sidebar, and see.

I have a rough idea of what the problem could be, but without a way to reproduce it would be difficult to track down what's going on. 

Since I gather that your code would eventually be public, can you reproduce this issue from a fresh start of VSCode and just post the log? See: https://github.com/dramforever/vscode-ghc-simple#debuggingissues

**Comment by agocorona** [comment]
Here is the log. The error is at the end:

```
ghci | ===ghci_barrier_0.7010069536037942===
    -> :info !!
ghci | (!!) :: [a] -> Int -> a 	-- Defined in ‘GHC.List’
ghci | 
ghci | ===ghci_barrier_0.43020809078398603===
    -> :m *undefined
ghci | syntax:  :module [+/-] [*]M1 ... [*]Mn
ghci | 
ghci | ===ghci_barrier_0.9692850287325567===
    -> :doc !!
ghci | unknown command ':doc'
ghci | use :? for help.
ghci | 
ghci | ===ghci_barrier_0.7906499339320436===
    -> :m *undefined
ghci | syntax:  :module [+/-] [*]M1 ... [*]Mn
ghci | 
ghci | ===ghci_barrier_0.990610457876105===
    -> :complete repl 50 "                           ,buff"
ghci | 0 0 "                           ,"
ghci | 
ghci | ===ghci_barrier_0.765719646570723===
    -> :set +c
    |> :reload
ghci | 
ghci | /projects/transient-stack/transient-universe/src/Transient/Move/Internals.hs:135:0: error:
ghci |      warning: extra tokens at end of #ifdef directive
ghci |      #ifdef MIN_VERSION_base(4,11,0)
ghci |      ^
ghci | [1 of 1] Compiling Transient.Move.Internals ( /projects/transient-stack/transient-universe/src/Transient/Move/Internals.hs, interpreted )
ghci | 
ghci | /projects/transient-stack/transient-universe/src/Transient/Move/Internals.hs:1874:42-43: error:
ghci |     parse error on input ‘::’
ghci | Failed, no modules loaded.
ghci | 
ghci | ===ghci_barrier_0.2330253134587572===
    -> :show modules
ghci | 
ghci | ===ghci_barrier_0.3168714860216637===
    -> :module
ghci | 
ghci | ===ghci_barrier_0.4528343795485634===
Error checking /projects/transient-stack/transient-universe/src/Transient/Move/Internals.hs: Error: Illegal argument: character must be non-negative
```

**Comment by dramforever** [comment]
Got it, should be fixed in v0.1.24, which should be released in a few minutes after it goes through CI.

**Comment by agocorona** [comment]
It works. Thanks!
